python/private/pypi/hub_repository.bzl

```diff
@@ -83,6 +84,10 @@ hub_repository = repository_rule(
 The list of packages that will be exposed via all_*requirements macros. Defaults to whl_map keys.
 """,
         ),
+        "platform_constraint_values": attr.string_list_dict(
```

Comment:
Why does a `{"platform name": [list of constraint labels]}` need to be passed?

When the hub generates selects, it just needs the matching constraints and the label they map to. The "human friendly" platform string isn't needed.

Reply:
SGTM. I don't mind the name being wired through for intelligibility. The main thing is I don't want the value to have real meaning (e.g. we could just replace them with platform_1, platform_2, etc).